### ISSUE TITLE
ajax cart was not working 

### ISSUE THREAD

**Issue by paval123** [trive-digital/Revo-ProductSliderM2#4]
trive-digital/Revo-ProductSliderM2  ajax add to cart was not working

**Comment by chombe** [comment]
Hi @paval123  need more information about your issue.
Did you enabled "Enable ajax add to cart" option from slider configuration?

This is [demo site](http://revo.trivedev.com/wayfarer-messenger-bag.html) where you can check...

**Comment by paval123** [comment]
Yeah, I am Set 'YES' to Enable ajax add to cart option in Trive > settings in admin panel

There is no issue on console or anyfile. When I am clicking the add cart button, the button is changed to adding...

Afterthat, there is no response. Please help me.

**Comment by chombe** [comment]
Those are general settings.. Can you please go to Trive -> Manage Sliders -> Edit on slider you created and check "Enable ajax add to cart" there.

**Comment by paval123** [comment]
That option also enabled, but still ajax add to cart was not working.

That demo site also AJAX ADD TO CART was not woking. Link: http://revo.trivedev.com/. Please help me.

**Comment by mombrea** [comment]
Same issue

**Comment by chombe** [comment]
Hi @paval123  

i just checked on my local 2.1.4 clean installation and on demo site at http://revo.trivedev.com/ and it's working fine. Maybe there are some conflicts on your installation with other extensions!!...
